pkg/registry/resolver.go

```diff
@@ -69,19 +76,41 @@ func (r *registry) Resolve(name string, config api.MultiStageTestConfiguration)
 	post, errs := r.process(config.Post, sets.NewString(), stack)
 	expandedFlow.Post = append(expandedFlow.Post, post...)
 	resolveErrors = append(resolveErrors, errs...)
-
+	for u := range rec.unused {
+		resolveErrors = append(resolveErrors, stackErrorf(stack, "no step declares parameter %q", u))
```

Comment:
So it is an error if I specify a parameter which is not explicitly consumed by any step? That sounds excessive... It also prevents us from e.g. adding a parameter to a step without breaking existing users...

Reply:
Don't we *want* to prevent those? The PR adding the parameter will fail validation if the tests are also not updated.

Reply:
I was worried about the `redhat-operator-ecosystem/release` content, but I guess you are right. And it has an advantage of actively stopping the possible confusion that items in `environment` are just propagated everywhere, even if the step/chain/workflow does not declare to have that parameter, which is a good thing.